features/fee_calculator/fixed_fee_calculator.feature

```diff
@@ -21,46 +21,22 @@ Feature: Advocate requests calculator
 
     Then I click "Continue" in the claim form
 
-    Given I insert the VCR cassette 'features/fixed_fee_calculations/agfs/scheme_nine'
+    Given I insert the VCR cassette 'features/fee_calculator/fixed_fee_calculator'
 
     And I select an advocate category of 'Junior alone'
-    And I add a fixed fee 'Appeals to the crown court against sentence'
+    And I add a fixed fee 'Appeals to the crown court against conviction'
     Then the last fixed fee case numbers section should not be visible
-    Then the last fixed fee rate should be populated with '108.00'
+    Then the last fixed fee rate should be populated with '130.00'
 
     And I add a fixed fee 'Number of cases uplift' with case numbers
     Then the last fixed fee case numbers section should be visible
-    Then the last fixed fee rate should be populated with '21.60'
+    Then the last fixed fee rate should be populated with '26.00'
 
-    And I add a fixed fee of defendants uplift
+    And I add a fixed fee 'Number of defendants uplift'
     Then the last fixed fee case numbers section should not be visible
-    Then the last fixed fee rate should be populated with '21.60'
+    Then the last fixed fee rate should be populated with '26.00'
```

Comment:
Add steps to exercise 
1. changing a parent quantity impact on uplift unit price
2. changing the advocate category recalculates all prices